.github/workflows/pull_requests.yml

```diff
@@ -31,3 +31,14 @@ jobs:
     secrets:
       access_key: "${{secrets.access_key_prod}}"
       secret_key: "${{secrets.secret_key_prod}}"
+  tfsec:
+    name: tfsec PR commenter
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@master
+      - name: tfsec
+        uses: tfsec/tfsec-pr-commenter-action@main
```

Comment:
Looks like `tfsec/tfsec-pr-commenter-action` just redirects to `aquasecurity/tfsec-pr-commenter-action` so I think it'd be clearer just to use the latter?

Reply:
Sounds sensible, done